alex/tools/kaldi/env_voip_cs.sh

```diff
@@ -39,5 +39,8 @@ export train_mmi_boost=0.05
 export mmi_beam=16.0
 export mmi_lat_beam=10.0
 
+export smbr_beam=16.0
+export smbr_lat_beam=10.0
```

Comment:
See http://homepages.inf.ed.ac.uk/aghoshal/pubs/icassp12-lattices.pdf Figure 2 why I think 6 is more than enough ;-)